scripts/score_with_katago.py

```diff
@@ -69,17 +69,24 @@ def main():
         help="Path to file to write output SGFs",
         default=os.devnull,
     )
+    tmp_dir = Path(f"/tmp/score-with-katago-{getpass.getuser()}")
```

Comment:
Using `with tempfile.TemporaryDirectory() as tmpdirname:` might be more robust (avoid race conditions if running two instances of this code on the same FS), but aware this line was already present just moved so perhaps this was already discussed in a previous PR?

Reply:
yeah I think Nora pointed out the same thing last time, I'll change to tempfile.TemporaryDirectory() but I'll do it in a separate PR since it'll shift the indent level up everywhere and cause a huge diff